packages/@aws-cdk/cloudformation-include/test/test-templates/parameter-references.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Transform" : {
+    "Name" : "AWS::Include",
```

Comment:
This makes no sense, right? This form is only allowed for `Fn::Transform`, not in the `Transform` section.

Reply:
That template passes verification; unless the docs here[https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/create-reusable-transform-function-snippets-and-add-to-your-template-with-aws-include-transform.html] are lying, this makes sense.

Reply:
Hmm, OK. We will need to update the `Transform` template section merging logic [here](https://github.com/aws/aws-cdk/blob/908dd69ec6e7d2037e299513e424d88921ebe04f/packages/%40aws-cdk/core/lib/stack.ts#L971-L972) then (in a separate change, of course).